2016-02-24 20:09:20,317 - [INFO] - from play in pool-4-thread-4 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2016-02-24 20:10:08,931 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 20:10:18,011 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2016-02-24 20:12:15,025 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 20:12:15,231 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2016-02-24 20:20:39,406 - [INFO] - from play in play-internal-execution-context-5 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 20:20:39,592 - [INFO] - from play in play-internal-execution-context-5 
Application started (Dev)

2016-02-24 20:46:05,243 - [ERROR] - from application in New I/O worker #2 


! @6p87kga1b - Internal server error, for (GET) [/my_profile.html] ->

play.PlayExceptions$CompilationException: Compilation error[error: cannot find symbol]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.6]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.6]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2016-02-24 20:48:19,045 - [ERROR] - from application in New I/O worker #3 


! @6p87ko002 - Internal server error, for (GET) [/my_profile.html] ->

play.PlayExceptions$CompilationException: Compilation error[error: cannot find symbol]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.6]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.6]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2016-02-24 20:52:32,284 - [INFO] - from play in play-internal-execution-context-5 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 20:52:32,540 - [INFO] - from play in play-internal-execution-context-5 
Application started (Dev)

2016-02-24 20:55:31,059 - [ERROR] - from play in play-internal-execution-context-5 
Cannot invoke the action, eventually got an error: javax.persistence.PersistenceException: ERROR executing DML bindLog[] error[Data truncation: Data too long for column 'post_content' at row 1]

2016-02-24 20:55:31,090 - [ERROR] - from application in play-internal-execution-context-5 


! @6p87lmf34 - Internal server error, for (POST) [/newpost_finish.html] ->

play.api.Application$$anon$1: Execution exception[[PersistenceException: ERROR executing DML bindLog[] error[Data truncation: Data too long for column 'post_content' at row 1]]]
	at play.api.Application$class.handleError(Application.scala:293) ~[play_2.10.jar:2.2.6]
	at play.api.DefaultApplication.handleError(Application.scala:399) [play_2.10.jar:2.2.6]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:268) [play_2.10.jar:2.2.6]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:268) [play_2.10.jar:2.2.6]
	at scala.Option.map(Option.scala:145) [scala-library.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:268) [play_2.10.jar:2.2.6]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:264) [play_2.10.jar:2.2.6]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.api.libs.iteratee.Execution$$anon$1.execute(Execution.scala:43) [play-iteratees_2.10.jar:2.2.6]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library.jar:na]
Caused by: javax.persistence.PersistenceException: ERROR executing DML bindLog[] error[Data truncation: Data too long for column 'post_content' at row 1]
	at com.avaje.ebeaninternal.server.persist.dml.DmlBeanPersister.execute(DmlBeanPersister.java:97) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.persist.dml.DmlBeanPersister.insert(DmlBeanPersister.java:57) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.persist.DefaultPersistExecute.executeInsertBean(DefaultPersistExecute.java:66) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.core.PersistRequestBean.executeNow(PersistRequestBean.java:448) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.core.PersistRequestBean.executeOrQueue(PersistRequestBean.java:478) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:335) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.persist.DefaultPersister.saveEnhanced(DefaultPersister.java:310) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.persist.DefaultPersister.saveRecurse(DefaultPersister.java:280) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.persist.DefaultPersister.save(DefaultPersister.java:248) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1568) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.core.DefaultServer.save(DefaultServer.java:1558) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebean.Ebean.save(Ebean.java:453) ~[avaje-ebeanorm.jar:na]
	at play.db.ebean.Model.save(Model.java:91) ~[play-java-ebean_2.10.jar:2.2.6]
	at controllers.Dashboard.newPostFinish(Dashboard.java:51) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$11$$anonfun$apply$11.apply(routes_routing.scala:165) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$11$$anonfun$apply$11.apply(routes_routing.scala:165) ~[na:na]
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183) ~[play_2.10.jar:2.2.6]
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377) ~[play_2.10.jar:2.2.6]
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56) ~[play_2.10.jar:2.2.6]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91) ~[play_2.10.jar:2.2.6]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90) ~[play_2.10.jar:2.2.6]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.6]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.6]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251) ~[scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249) ~[scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37) ~[play_2.10.jar:2.2.6]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42) ~[akka-actor_2.10.jar:2.2.0]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386) ~[akka-actor_2.10.jar:2.2.0]
	... 4 common frames omitted
Caused by: com.mysql.jdbc.MysqlDataTruncation: Data truncation: Data too long for column 'post_content' at row 1
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3885) ~[mysql-connector-java-5.1.34.jar:5.1.34]
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3823) ~[mysql-connector-java-5.1.34.jar:5.1.34]
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2435) ~[mysql-connector-java-5.1.34.jar:5.1.34]
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2582) ~[mysql-connector-java-5.1.34.jar:5.1.34]
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2530) ~[mysql-connector-java-5.1.34.jar:5.1.34]
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1907) ~[mysql-connector-java-5.1.34.jar:5.1.34]
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2141) ~[mysql-connector-java-5.1.34.jar:5.1.34]
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2077) ~[mysql-connector-java-5.1.34.jar:5.1.34]
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2062) ~[mysql-connector-java-5.1.34.jar:5.1.34]
	at com.jolbox.bonecp.PreparedStatementHandle.executeUpdate(PreparedStatementHandle.java:205) ~[bonecp.jar:na]
	at com.avaje.ebeaninternal.server.type.DataBind.executeUpdate(DataBind.java:55) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.persist.dml.InsertHandler.execute(InsertHandler.java:134) ~[avaje-ebeanorm.jar:na]
	at com.avaje.ebeaninternal.server.persist.dml.DmlBeanPersister.execute(DmlBeanPersister.java:86) ~[avaje-ebeanorm.jar:na]
	... 32 common frames omitted

2016-02-24 20:59:26,432 - [INFO] - from play in play-internal-execution-context-5 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 20:59:26,722 - [INFO] - from play in play-internal-execution-context-5 
Application started (Dev)

2016-02-24 21:05:10,196 - [INFO] - from play in play-internal-execution-context-16 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 21:05:10,365 - [INFO] - from play in play-internal-execution-context-16 
Application started (Dev)

2016-02-24 21:07:37,271 - [INFO] - from play in play-internal-execution-context-16 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 21:07:37,448 - [INFO] - from play in play-internal-execution-context-16 
Application started (Dev)

2016-02-24 21:11:06,035 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 21:11:06,288 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2016-02-24 21:15:35,130 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 21:15:35,348 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2016-02-24 21:17:19,318 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 21:17:19,521 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2016-02-24 21:18:46,988 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 21:18:47,174 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2016-02-24 21:19:55,327 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 21:19:55,526 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2016-02-24 21:21:09,689 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 21:21:09,886 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2016-02-24 21:23:41,115 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 21:23:41,281 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2016-02-24 21:24:07,175 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 21:24:07,343 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2016-02-24 21:25:08,455 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 21:25:08,634 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

2016-02-24 21:25:52,402 - [INFO] - from play in play-internal-execution-context-21 
database [default] connected at jdbc:mysql://localhost:3306/play_app_1

2016-02-24 21:25:52,561 - [INFO] - from play in play-internal-execution-context-21 
Application started (Dev)

